py3status/modules/wifi.py

```diff
@@ -73,6 +73,8 @@ class Py3status:
     signal_bad = 29
     signal_degraded = 49
     use_sudo = False
+    _ssid = None
+    _max_bitrate = 0
```

Comment:
I'm sorry this isn't allowed in modules, only configuration parameters are allowed here.

I'm interested in your error as it looks like something is seriously screwed.  Can you provide the info on exactly what you were running eg
```
2017-04-11 11:47:00 INFO ========
2017-04-11 11:47:00 INFO Starting py3status version 3.6rc1 python 2.7.13
2017-04-11 11:47:00 INFO git branch: fix-group-width-issue
2017-04-11 11:47:00 INFO git commit: c19ad18 fix group width issue when no output
```
else it is hard to interpret the traceback here